lib/clean.js

```diff
@@ -1,18 +1,18 @@
 'use strict';
 
-var merge = require('lodash').merge;
-var del = require('del');
+import {merge} from 'lodash';
+import del from 'del';
 
-var defaults = {
+const defaults = {
   dest: 'dist'
 };
 
-module.exports = function (gulp, options) {
+module.exports = (gulp, options) => {
   var opts = merge({}, defaults, options);
   var dest = opts.dest;
 
-  gulp.task('clean', function (done) {
-    del(dest).then(function () {
+  gulp.task('clean', (done) => {
```

Comment:
**Stylistic nitpick ahead** (feel free to ignore):

The parenthesis are only required around the parameters when there are multiple, so this should work too:

```js
gulp.task('clean', done => {
  done();
});
```

Also, the brackets are only required when the function is more than one line, so this should work as well:

```js
gulp.task('clean', done => done());
```

Note: when used in this manner, the return value of `done()` will be returned and sent to the `done` callback, though this should have no effect on this particular usage (let the tests verify that.)